qutip/tests/test_qobj.py

```diff
@@ -1014,3 +1016,26 @@ def test_unit():
     psi.unit(inplace=True)
     assert psi == psi2
     assert np.allclose(np.linalg.norm(psi.full()), 1.0)
+
+
+def test_tidyup():
+    small = Qobj(1e-15)
+    small.tidyup(1e-14)
+    assert small.norm() == 0
+
+
+@contextmanager
+def tidyup_tol(tol):
```

Comment:
```suggestion
def auto_tidyup_tol(tol):
```
Thought it might be nice to make it a bit clearer what the context manager changed.